riptide-stream/src/main/java/org/zalando/riptide/stream/Streams.java

```diff
@@ -56,23 +56,50 @@
  */
 public final class Streams {
 
+    /** Default stream character encoding. */
     public static final Charset DEFAULT_CHARSET = Charset.forName("UTF-8");
```

Comment:
Is it necessary to have this as a public API? Also, you can just use StandardCharsets.UTF_8